### PR TITLE
RHMAP-6647 - Tweaks for OSE template

### PR DESCRIPTION
**Pull request by jasonmadigan**
Tested with OSE3.2. See also https://github.com/fheng/fh-openshift-templates/pull/216 for OpenShift 3 template.

Some tweaks:

* Renamed the mysql service to `mysql` rather than `unifiedpush` so that the image works OOTB with the  OS MySQL Ephemeral / Persistant templates

### CODE REVIEW

File: wildfly/unifiedpush-wildfly/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Use latest aerogear/wildfly image as the base
-FROM aerogear/wildfly
+FROM rhmap/wildfly
```
Comment (jasonmadigan) [inline wildfly/unifiedpush-wildfly/Dockerfile]: We can switch this back when we build and push the `aerogear/wildfly` image and tag it as 1.1.x or similar (the one on master doesn't seem to start correctly with the 1.1.x branch version of UPS).

**Comment by jasonmadigan** [comment]
@matzew FYI

**Comment by matzew** [comment]
@jasonmadigan not sure we should do this in upstream

**Comment by jasonmadigan** [comment]
@matzew yeah, I don't really mind adding this to `fh-dockerfiles` standalone. Making changes now to use the https://github.com/fheng/aerogear-unifiedpush-server/commits/ups_rhmap_3.12.x tree + health/rediness checks, which makes even less sense upstream.

**Pull request closed without merging**